azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/localuser/LocalParticipantViewModel.kt

```diff
@@ -60,7 +60,8 @@ internal class LocalParticipantViewModel(
         val displayVideo = shouldDisplayVideo(videoStreamID)
         val displayLobbyOverlay = shouldDisplayLobbyOverlay(callingState)
         val displayFullScreenAvatar =
-            shouldDisplayFullScreenAvatar(displayVideo, displayLobbyOverlay, viewMode)
+            shouldDisplayFullScreenAvatar(displayVideo, displayLobbyOverlay, viewMode) &&
+                callingState == CallingStatus.CONNECTED
```

Comment:
This condition `callingState == CallingStatus.CONNECTED` should go to the `shouldDisplayFullScreenAvatar`
Otherwise you put logic in two places